library/src/androidTest/java/net/grandcentrix/tray/provider/TrayProviderHelperTest.java

```diff
@@ -54,6 +58,40 @@
 
     private TrayUri mTrayUri;
 
+    public void testCursorToTrayItem() throws Exception{
+
+        final MatrixCursor matrixCursor = new MatrixCursor(new String[]{
+                TrayContract.Preferences.Columns.KEY,
+                TrayContract.Preferences.Columns.VALUE,
+                TrayContract.Preferences.Columns.MODULE,
+                TrayContract.Preferences.Columns.CREATED,
+                TrayContract.Preferences.Columns.UPDATED,
+                TrayContract.Preferences.Columns.MIGRATED_KEY
+        });
+        final Date created = new Date();
+        final Date updated = new Date();
+        matrixCursor.addRow(new Object[]{
+                "key",
+                "value",
+                "module",
+                created.getTime(),
+                updated.getTime(),
+                "migratedKey"
+        });
+        assertTrue(TextUtils.isEmpty(""));
+        assertFalse(TextUtils.isEmpty("asdf"));
```

Comment:
I assume the [TextUtils](http://developer.android.com/reference/android/text/TextUtils.html) to work correctly